### DISCUSSION TITLE
No Claude 3.5 Sonnet in VSCode

### DISCUSSION THREAD

**Discussion by maple0leaves** [community/community#143672] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body
As shown in the title，my vscode has not claude3.5 sonnet,why? When I first time install copilot, it‘s still here,but when I open vscode again,it disappeared.
![image](https://github.com/user-attachments/assets/fe26f48a-fd8e-49c6-a4bb-8b3314f019ba)
![image](https://github.com/user-attachments/assets/d5a1db00-557c-4460-8675-96650c6c2718)

**Comment by InWamos** [comment]
Have you updated the copilot extension? It is available for me on linux on the last extension version
![image](https://github.com/user-attachments/assets/aff1d7f2-9ae7-40ae-a988-eb72b4d5d1e8)

**Reply by maple0leaves** [reply]
thank you ,but it is last extension version,hhhh,so strange

**Reply by maple0leaves** [reply]
I try to install  it again

**Reply by InWamos** [reply]
Make sure you have the following versions of those extensions:
# GitHub Copilot Chat - v0.22.1
# GitHub Copilot - v1.243.0

**Reply by maple0leaves** [reply]
still no

**Reply by maple0leaves** [reply]
yes ,my version is same as you ,mybe problem is my vpn or some other

**Reply by InWamos** [reply, marked as answer]
> yes ,my version is same as you ,mybe problem is my vpn

https://docs.github.com/en/copilot/using-github-copilot/using-claude-sonnet-in-github-copilot#configuring-access

try reading this

And pay attention to this line:
You can enable the model directly in your personal settings on the GitHub website. See "[Managing Copilot policies as an individual subscriber](https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-copilot-policies-as-an-individual-subscriber#enabling-or-disabling-claude-35-sonnet)."

**Reply by maple0leaves** [reply]
Ok thank you very much

**Reply by DullJZ** [reply]
> yes ,my version is same as you ,mybe problem is my vpn or some other

That's it. When I'm using HK proxy, it disappeared. As I restart VSCode with Taiwan proxy on, it appeared.

**Reply by zsx22705** [reply]
That's it. When I'm using SG proxy, it disappeared. As I restart VSCode with JP proxy on, it appeared.

**Reply by InWamos** [reply]
> Ok thank you very much

Have you solved your problem? Mark my message as the answer in that case please

**Reply by maple0leaves** [reply]
Yes ,thank you!

**Reply by InWamos** [reply]
> Yes ,thank you!

Great! Could you close the tread by marking my reply as the answer? This would be much appreciated

**Comment by ivanberry** [comment]
what's the rule of your VPN? githubcopilot or github.com

**Reply by ivanberry** [reply]
bypass githubcopilot domain keyword with the VPN rule
